src/main/menus/edit.js

```diff
@@ -88,5 +88,20 @@ export default {
         actions.insertImage(browserWindow, 'upload')
       }
     }]
+  }, {
+    type: 'separator'
+  }, {
+    label: 'Line Ending',
+    submenu: [{
+      label: 'Carriage return and line feed (CRLF)',
```

Comment:
'Carriage return and line feed (CRLF)' and 'Line feed (LF)' are two opposite menu item. So I think these two menu item should be type `radio`, in others words, you can only select one of them.

like this bellow:

```javascript
label: 'Carriage return and line feed (CRLF)',
type: 'radio'
// other codes
```

this will add a `checked marker` at the beginning of the selected menu item.

Another thing is.

The selected status should also be the identity of the line break of the current edit document in active window.

For example, if the current edit file is `LF` and the `Line feed (LF)` menu item should be selected. and I changed to another `CRLF` file and the `'Carriage return and line feed (CRLF)'` menu item should be selected.

Reply:
Cool idea. I created a new commit, but this may have side effects when another Mark Text window is selected before a newly created window is loaded. But it's just UI, so don't care.